### DISCUSSION TITLE
Welcome to Aam Digital's Community Forum!

### DISCUSSION THREAD

**Discussion by sleidig** [Aam-Digital/ndb-core#1078] [General]
Welcome! It's great that you are interested in or working with Aam Digital.

These Github Discussons here are a good place to ask questions, discuss ideas or provide feedback. Simply create a new discussion thread. We are looking forward to hearing from you! 🙂

**Comment by helloaashu** [comment]
Okay i would love to do that

**Reply by sleidig** [reply]
Hi @Aashutoshbro thanks for posting! What are you interested in around Aam Digital?
